selenium_tests/course_info/course_info_is_search_page_loaded_test.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from selenium_tests.course_info.course_info_base_test_case import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.course_info_parent_page_object import CourseInfoParentPage
+from selenium_tests.course_info.page_objects.course_info_search_page_object import CourseSearchPageObject
+
+
+class CourseInfoIsSearchPageLoadedTest(CourseInfoBaseTestCase):
+
+    def test_is_loaded(self):
+        """
+        Verify  that the course info search page is loaded by checking against page title
+        """
+        driver = self.driver
```

Comment:
I would prefer to see all of the `driver = self.driver` lines removed (in the other tests too), and `self.driver` used in place of `driver`. Tightens things up a bit.